framework/math/quadratures/gauss_quadrature.cc

```diff
@@ -1,56 +1,42 @@
-#include "framework/math/quadratures/quadrature.h"
-
+#include "framework/math/quadratures/gauss_quadrature.h"
 #include "framework/logging/log_exceptions.h"
 
 namespace opensn
 {
 
 InputParameters
-Quadrature::GetInputParameters()
+GaussQuadrature::GetInputParameters()
 {
   InputParameters params = Object::GetInputParameters();
 
   params.SetGeneralDescription("\\defgroup math__Quadrature\n"
                                "\\ingroup LuaQuadrature\n"
-                               "Base class for 1D quadratures");
+                               "Base class for Gauss quadratures");
 
   params.AddRequiredParameter<int>("order", "Quadrature order.");
-  params.AddOptionalParameter("verbose", false, "Enables verbose operations");
 
-  params.ConstrainParameterRange("order", AllowableRangeLowHighLimit::New(0, 43));
+  params.AddOptionalParameter("verbose", false, "Enables verbose operations");
 
   return params;
 }
 
-Quadrature::Quadrature(const InputParameters& params)
-  : Object(params),
-    order_(static_cast<QuadratureOrder>(params.GetParamValue<int>("order"))),
-    verbose_(params.GetParamValue<bool>("verbose"))
-{
-}
-
 void
-Quadrature::SetRange(const std::pair<double, double>& range)
+GaussQuadrature::SetRange(const std::pair<double, double>& range)
 {
   const auto& old_range = range_;
   const auto& new_range = range;
-
   const double h_new = new_range.second - new_range.first;
   const double h_old = old_range.second - old_range.first;
 
-  OpenSnInvalidArgumentIf(h_new <= 0.0 or h_old <= 0.0, "Called with negative or zero ranges.");
-
+  // OpenSnInvalidArgumentIf(h_new <= 0.0 or h_old <= 0.0, "Called with negative or zero ranges.");
```

Comment:
Is this a left over from debugging? If not, then remove this like

Reply:
Good catch! I had disabled that for testing.  Fixed!